### PR TITLE
Pdf poc

### PR DESCRIPTION
**Pull request by Stefafrontani**
Integrate router with my changes.
Now when we click _Create Doc_ some buttons are displayed in order to See or Download a pdf

image

### CODE REVIEW

File: notes.txt [hunk 1]
```diff
@@ -0,0 +1,117 @@
+## Module Resolution
```
Comment (globantPipo) [inline notes.txt:1]: We should just include this into the readme as a Tech notes, or the README.md in SRC folder. So is accesible from github

File: src/components/Nav/index.tsx [hunk 2]
```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 // eslint-disable-next-line no-unused-vars
-import { BrowserRouter as Router, Link } from 'react-router-dom';
+import { Link } from 'react-router-dom';
```
Comment (globantPipo) [inline src/components/Nav/index.tsx:3]: Is this still working ? I'm just mentioning it cause in the past, I needed `Router` obj in this context to make `Link` work.

File: src/containers/CreateDoc/pdf.ts [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    sOne: {
+        content: 'Content',
+        header: 'Header',
+        footer: 'Footer',
+        background: 'background'
+    }
+}
+
+function createPdf(pdfType: string = 'default'): TCreatedPdf {
+    return PdfMake.createPdf(pdfDefinitions[pdfType]);
```
Comment (globantPipo) [inline src/containers/CreateDoc/pdf.ts:25]: Are `open` and `download` function inside the obj return of this function ? 

---------

Anyway, we should try to send the inner text to this module, from the createDoc `index` so we start wiring some things up between the modules.

File: src/containers/CreateDoc/pdf.ts [hunk 4]
```diff
@@ -0,0 +1,28 @@
+import PdfMake from 'pdfmake/build/pdfmake';
+
+import { TDocumentDefinitions } from 'types';
+import { TCreatedPdf } from 'types';
+
+interface PdfDefinitions {
+    [pdfDefinition: string]: TDocumentDefinitions
+}
+
+const pdfDefinitions: PdfDefinitions = {
```
Comment (globantPipo) [inline src/containers/CreateDoc/pdf.ts:10]: Not sure if I understand this, `pdfDefinitions` is the interface for the const `pdfDefinitions` (we need to adjust those names). 

Then the interface has a SINGLE optional key `pdfDefinition` which should have a `TDocumentDefinitions`, but here in the const, we have 2 keys.

How does it work ?

File: src/components/Nav/index.tsx (hunk 2 above)
Comment (Stefafrontani) [reply src/components/Nav/index.tsx:3]: Yes. Still working

File: notes.txt (hunk 1 above)
Comment (Stefafrontani) [reply notes.txt:1]: Think it's a good idea 👌 . Let's define in today's daily where we want to place this :) :)

Comment (globantPipo) [reply notes.txt:1]: So the right place is going to be the same readme, let-s keep it simple : )

File: src/components/Nav/index.tsx (hunk 2 above)
Comment (globantPipo) [reply src/components/Nav/index.tsx:3]: Wonder why... Anyway ty : )

File: src/containers/CreateDoc/pdf.ts (hunk 3 above)
Comment (globantPipo) [reply src/containers/CreateDoc/pdf.ts:25]: Fair enough

File: notes.txt (hunk 1 above)
Comment (Stefafrontani) [reply notes.txt:1]: 👌 Great 😄 😄

**Comment by globantPipo** [comment]
We can addres the change in the readme whenever you want @Stefafrontani . Let me know whenever you want me to merge this.

**Pull request merged**